cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/MockCruiseControl.java

```diff
@@ -207,6 +207,24 @@ public void setupCCRebalanceNotEnoughDataError(CruiseControlEndpoints endpoint)
                                 .withBody(jsonError)
                                 .withHeaders(header(USER_TASK_ID_HEADER, REBALANCE_NOT_ENOUGH_VALID_WINDOWS_ERROR_RESPONSE_UTID))
                                 .withDelay(TimeUnit.SECONDS, 0));
+
+        // Rebalance response with no goal that returns an error
+        server
+                .when(
+                        request()
+                                .withMethod("POST")
+                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.JSON.toString(), "true"))
+                                .withQueryStringParameter(Parameter.param(CruiseControlParameters.DRY_RUN.toString(), "true|false"))
+                                .withQueryStringParameter(buildBrokerIdParameter(endpoint))
+                                .withPath(endpoint.toString())
+                                .withHeader(AUTH_HEADER)
+                                .withSecure(true))
+                .respond(
+                        response()
+                                .withStatusCode(500)
+                                .withBody(jsonError)
+                                .withHeaders(header(USER_TASK_ID_HEADER, REBALANCE_NOT_ENOUGH_VALID_WINDOWS_ERROR_RESPONSE_UTID))
+                                .withDelay(TimeUnit.SECONDS, 0));
```

Comment:
I see this mocking the same as the above but missing VERBOSE only (maybe not supported by remove-disks endpoint?) but then `.withQueryStringParameter(buildBrokerIdParameter(endpoint))` which as you can see add broker id parameters for the add and remove brokers so which don't make any sense for remove disks, or?